src/components/AddTodo.jsx

```diff
@@ -0,0 +1,55 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+
+
+const AddTodo = ({ addTodo }) => {
```

Comment:
I think is confusing since the component name and the prop being passed have the same name!